client/analytics/settings/historical-data/layout.js

```diff
@@ -119,88 +107,86 @@ class HistoricalDataLayout extends Component {
 }
 
 export default withSelect( ( select, props ) => {
-	const {
-		getImportStatus,
-		isGetImportStatusRequesting,
-		getImportTotals,
-	} = select( 'wc-api' );
+	const { getImportError, getImportStatus, getImportTotals } = select(
+		IMPORT_STORE_NAME
+	);
 	const {
 		activeImport,
+		cacheNeedsClearning,
 		dateFormat,
-		lastImportStartTimestamp,
-		lastImportStopTimestamp,
+		inProgress,
 		onImportStarted,
 		onImportFinished,
 		period,
+		startStatusCheckInterval,
 		skipChecked,
 	} = props;
 
-	const inProgress =
-		( typeof lastImportStartTimestamp !== 'undefined' &&
-			typeof lastImportStopTimestamp === 'undefined' ) ||
-		lastImportStartTimestamp > lastImportStopTimestamp;
-
 	const params = formatParams( dateFormat, period, skipChecked );
-	// Use timestamp to invalidate previous totals when the import finished/stopped
-	const { customers, orders } = getImportTotals(
-		params,
-		lastImportStopTimestamp
+	const { customers, orders, lastImportStartTimestamp } = getImportTotals(
+		params
 	);
-	const requirement = inProgress
-		? {
-				freshness: 3 * SECOND,
-				timeout: 3 * SECOND,
-		  }
-		: DEFAULT_REQUIREMENT;
 
-	// Use timestamp to invalidate previous status when a new import starts
 	const {
 		customers: customersStatus,
 		imported_from: importDate,
 		is_importing: isImporting,
 		orders: ordersStatus,
-	} = getImportStatus( lastImportStartTimestamp, requirement );
+	} = getImportStatus( lastImportStartTimestamp );
 	const { imported: customersProgress, total: customersTotal } =
 		customersStatus || {};
 	const { imported: ordersProgress, total: ordersTotal } = ordersStatus || {};
-	const isStatusLoading = isGetImportStatusRequesting(
-		lastImportStartTimestamp
+
+	const isError = Boolean(
+		getImportError( lastImportStartTimestamp ) ||
```

Comment:
is this `a || a`?

Reply:
Nice catch! This was fixed in the commit e417088